packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -415,6 +433,7 @@ export default {
         },
 
         processOrderNotFulfillable (eventData) {
+            this.handleCheckoutIssues();
```

Comment:
Can you add a test to `processOrderNotFulfillable` to cover this change please? Sorry just noticed this after approving.